bftengine/src/bftengine/ReplicaImp.hpp

```diff
@@ -474,9 +478,18 @@ class ReplicaImp : public InternalReplicaApi, public ReplicaForStateTransfer {
         std::make_shared<Recorder>(1, MAX_VALUE_NANOSECONDS, 3, concord::diagnostics::Unit::NANOSECONDS);
     std::shared_ptr<Recorder> onTransferringCompleteImp =
         std::make_shared<Recorder>(1, MAX_VALUE_NANOSECONDS, 3, concord::diagnostics::Unit::NANOSECONDS);
+
+    // Only updated by the primary
+    std::shared_ptr<Recorder> consensus =
+        std::make_shared<Recorder>(1, MAX_VALUE_MILLISECONDS, 3, concord::diagnostics::Unit::MILLISECONDS);
```

Comment:
I would move to MICROSECONDS here

Reply:
I can do that, but I'm not sure how much it helps us in practice.  Practically speaking, I didn't think we could get microsecond resolution. Even if we see `0`, we just know that consensus latency is not a problem. In production we have multiple messages traversing a real network, and with batching they are going to be fairly large. We are going to just end up discarding the last 3 digits. I noticed this problem where I used nanoseconds instead of micros elsewhere, and for most of those I want to change to micros, as anything less is unlikely or just a blip.  If we consistently see `0` we can change to microseconds.

Reply:
I changed it to MICROSECONDS.